File: conexion.php
<?php
$host = 'TuPasswordSeguro';
$user = 'root';          // tu usuario MySQL
$pass = '';     // tu contraseña
$db   = 'tiendademascotas';

$conn = new mysqli($host, $user, $pass, $db);
if ($conn->connect_error) {
    die("Conexión fallida: " . $conn->connect_error);
}
?>

File: conexion.py
import mysql.connector

try:
    conexion = mysql.connector.connect(
        host="localhost",
        user="root",
        password="",
        database="datadeanimales",
    )
    cursor = conexion.cursor()
    cursor.execute("SELECT * FROM datadeanimales")
    for fila in cursor.fetchall():
        print(fila)
except mysql.connector.Error as err:
    print("Error de conexión:", err)
finally:
    try:
        if 'conexion' in locals() and conexion.is_connected():
            conexion.close()
    except NameError:
        pass
